cityofaustin/atd-moped
# ETL process to synchronize Moped project data to Knack's Data Tracker

## Associated issues

This PR intends to close issue [# 7568](https://github.com/cityofaustin/atd-data-tech/issues/7568).

## Testing
Testing should be conducted on your local machine.

1) Review the README.md for instructions
2) Build the ETL image locally.
3) Start the Moped database and web interface locally.
4) Make changes to one of the fields which are pushed to Knack. 
5) Synchronize the project to Knack, if needed.
5) Inside the running ETL container, execute the `moped_data-tracker_sync_projects.py` script.
6) Observe the corollary field in the Knack App has been updated for that row.

---
#### Ship list
